.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version-file: go.mod
+          go-version: "1.23"
```

Comment:
Once a new version of Go 1.23 is released, `setup-go` will install that. When the jobs start running, `go` will see the `toolchain` directive and download that version instead. This is inefficient.

Probably okay for now, can you create a tracking issue internally to go back to `go-version-file` once a fix for https://github.com/actions/setup-go/issues/457 lands.

Reply:
That would be a reasonable workaround for now, but sounds very annoying to add everywhere tbh.